myapp/app/controllers/task_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
     @task.user_id = 1
     @task.user_type = 'Task'
```

Comment:
Please add a comment about why `user_id` and `user_type` is assigned here

Reply:
added